internal/node/node.bzl

```diff
@@ -167,6 +156,9 @@ _NODEJS_EXECUTABLE_ATTRS = {
         # dependency on a package like @bazel/typescript.
         # See discussion: https://github.com/bazelbuild/rules_typescript/issues/13
         default = Label("@//:node_modules")),
+    "node_modules_list": attr.label_list(
```

Comment:
could you add more docs here explaining why we need this additonal attribute? feels like it's making the surface area more complex for users.